### PR TITLE
anonymous user is redirected to login when creating a draft

### PR DESCRIPTION
**Pull request by vinyll**
the 2 broken tests are fixed in https://github.com/cafebabel/cafebabel.com/pull/111

### CODE REVIEW

File: cafebabel/articles/drafts/views.py [hunk 1]
```diff
@@ -17,8 +17,8 @@ def list():
 
 
 @drafts.route('/new/', methods=['get', 'post'])
-@editor_required
 @login_required
+@editor_required
```
Comment (davidbgk) [inline cafebabel/articles/drafts/views.py:21]: Not fond of the fact that order matters here, maybe add automagically `login_required` when `editor_required` is added? It looks more consistent to me.

Comment (vinyll) [reply cafebabel/articles/drafts/views.py:21]: What is not found?
I'm ok to have `login_required` embedded into `editor_required` in a different PR for keeping its original purpose on this one.

Comment (davidbgk) [reply cafebabel/articles/drafts/views.py:21]: Without the U? https://www.vocabulary.com/dictionary/fond

OK to put it in a subsequent PR

**Review by davidbgk — APPROVED** [review]

File: cafebabel/articles/drafts/views.py (hunk 1 above)
Comment (vinyll) [reply cafebabel/articles/drafts/views.py:21]: Issue [opened here](https://github.com/cafebabel/cafebabel.com/issues/114) for memory.

**Pull request merged**